.github/workflows/tests.yml

```diff
@@ -29,10 +29,55 @@ env:
   PYTHON_VERSION: "3.10"
   TF_PLUGIN_CACHE_DIR: "/home/runner/.terraform.d/plugin-cache"
   TF_VERSION: 1.4.4
+  TFTEST_COPY: 1
 
 jobs:
+  setup-tf-providers:
+    runs-on: ubuntu-latest
```

Comment:
can't you put this in the composite action? there's also a duplicate step

Reply:
Not really, we need the other jobs to run after we compute, fetch and cache all the providers.